fluid-project/eleventy-plugin-fluid
# chore(deps): update to FLUID-6145 branch of Infusion

Updates UIO integration provided by this plugin to use the FLUID-6145 line of Infusion.

Steps to test:

1. Check out this branch.
2. `npm install && npm start`.
3. Open http://localhost:8080.
4. Verify that UI Options adjusters work as expected.
